proj-xcode/PowerAuth2/PowerAuthSDK.h

```diff
@@ -348,6 +349,59 @@
  */
 @property (nonatomic, strong, nullable, readonly) PowerAuthActivationStatus * lastFetchedActivationStatus;
 
+/**
+ Start the protocol upgrade process.
+ 
+ @param password Required core password instance used to authenticate the protocol upgrade start.
+ @param newBiometryKek Optional parameter. If a biometry factor is configured in the current protocol version
+ and no new biometry KEK is provided, one will be automatically generated for the upgraded protocol.
+ @param callback A callback called when the upgrade task finishes.
+ @return Protocol upgrade task instance.
+ */
+- (nullable id<PowerAuthOperationTask>) startProtocolUpgradeWithCorePassword:(nonnull PowerAuthCorePassword*)password
+                                                          withNewBiometryKek:(nullable PowerAuthCoreData*)newBiometryKek
+                                                                    callback:(nonnull void(^)(PowerAuthProtocolUpgradeResult * _Nullable result, NSError * _Nullable error))callback
+                            NS_SWIFT_NAME(startProtocolUpgrade(password:withNewBiometryKek:callback:));
```

Comment:
Please rename `newBiometryKek` to `customBiometryKek`, to match naming used in `addBiometryFactor()` function.